README.md

```diff
@@ -55,3 +55,28 @@ oauth2 allow association change:	true
 ## Run
 
 Run the bot with `npm start`.
+
+
+## Background: Data flow
+
+This bot implements a minimal variant of [oauth2](https://en.wikipedia.org/wiki/OAuth#OAuth_2.0), an <q>open standard for access delegation</q>.
+
+In order to be transparent for admins and users, here's a short explanation of what that means:
+
+1. When you click on "Login with Delta Chat", your Browser is redirected to this bot (to `/authorize`).
+2. After this bot verified your email address, it hands your browser a token and redirects it back to discourse.
+3. Your discourse instance takes this token, it's `client_id`, and it's `client_secret`, and itself sends a request to this bot (`/token`).
```

Comment:
```suggestion
3. Your discourse instance takes this token, its `client_id`, and its `client_secret`, and itself sends a request to this bot (`/token`).
```

grammar: in this context, it's "its" and not "it's", as "it's" is short for "it is" and not for its counterpart "its".